pkgs/applications/audio/ncpamixer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, ncurses, libpulseaudio, pkgconfig }:
+
+stdenv.mkDerivation rec {
+
+  name = "ncpamixer-${version}";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "fulhax";
+    repo = "ncpamixer";
+    rev = version;
+    sha256 = "01kvd0pg5yraymlln5xdzqj1r6adxfvvza84wxn2481kcxfral54";
+  };
+
+  buildInputs = [ ncurses libpulseaudio ];
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  configurePhase = ''
+    make PREFIX=$out
```

Comment:
While this works I think the proper approach would be:

```patch
--- a/pkgs/applications/audio/ncpamixer/default.nix
+++ b/pkgs/applications/audio/ncpamixer/default.nix
@@ -16,7 +16,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake pkgconfig ];

   configurePhase = ''
-    make PREFIX=$out
+    make PREFIX=$out build/Makefile
+  '';
+
+  buildPhase = ''
+    make build
   '';

   meta = with stdenv.lib; {
```

Otherwise you are building the project in the configure phase.